test/models/vote_test.rb

```diff
@@ -0,0 +1,59 @@
+require "test_helper"
+
+describe Vote do
+
+  before do
+    @arbitrary_vote = Vote.new(user: User.find_by(username: "first-name123"), work: Work.find_by(category: 'movie'))
```

Comment:
Again, why not start with a valid vote defined in the fixtures? You follow the pattern of have a reference to a vote that is valid and then in the tests themselves, to test that they're invalid, you set them attribute by attribute to something invalid. So why not start with a valid fixture?